Fix fixation detector docs (#323)
pupil-labs/pupil-docs: src/core/terminology.md

cameras. It is recommended to run the [Camera Intrinsics Estimation](/core/software/pupil-capture/#camera-intrinsics-estimation)
for your Pupil Core _world_ camera after receiving it. Each camera is slightly different
and running the estimation locally will result in slightly more precise gaze mapping.

## Fixations

Salvucci and Goldberg define different categories of fixation detectors. One of them describes dispersion-based algorithms:

> I-DT identifies fixations as groups of consecutive points within a particular dispersion, or maximum separation. Because fixations typically have a duration of at least 100 ms, dispersion-based identification techniques often incorporate a minimum duration threshold of 100-200 ms to help alleviate equipment variability.
> 
> *--- Salvucci, D. D., & Goldberg, J. H. (2000, November). Identifying fixations and saccades in eye-tracking protocols. In Proceedings of the 2000 symposium on Eye tracking research & applications (pp. 71-78). ACM.*

> Pupil’s fixation detectors implement such dispersion-based algorithms. The exact procdure differs depending on whether fixations are detected in an online or offline context:

Reviewer: ```suggestion
Pupil Core's fixation detectors implement a dispersion-based method. The exact procedure differs depending on whether fixations are detected in an online or offline context:
```

### Online Fixation Detection

> In Pupil Capture, fixations are detected based on a dispersion threshold in terms of degrees of visual angle with a minimum duration. Fixation are published as soon as they comply with the constraints (dispersion and duration). This might result in a series of overlapping fixations.

Reviewer: ```suggestion
In Pupil Capture, fixations are detected based on a dispersion threshold in terms of degrees of visual angle with a minimum duration. Fixations are published as soon as they comply with the constraints (dispersion and duration). This might result in a series of overlapping fixations.
```
